### PR TITLE
Things i found

### PR DESCRIPTION
**Pull request by jdalsem**

### CODE REVIEW

File: views/default/river/elements/responses.php [hunk 1]
```diff
@@ -33,7 +33,7 @@
 		'subtype' => 'comment',
 		'container_guid' => $object->getGUID(),
 		'limit' => 3,
-		'order_by' => 'e.time_created desc',
+		'order_by' => [new OrderByClause('time_created', 'DESC')],
```
Comment (jeabakker) [inline views/default/river/elements/responses.php]: Missing a use statement, so this will probably crash

**Review by jeabakker — CHANGES_REQUESTED** [review]

File: views/default/river/elements/responses.php (hunk 1 above)
Comment (jdalsem) [reply views/default/river/elements/responses.php]: fixed

Comment (hypeJunction) [reply views/default/river/elements/responses.php]: Also missing in the first file, I think

Comment (jdalsem) [reply views/default/river/elements/responses.php]: no, was already present there

**Review by jeabakker — APPROVED** [review]

**Pull request merged**